lhotse/cut.py

```diff
@@ -1260,6 +1262,56 @@ def perturb_volume(self, factor: float, affix_id: bool = True) -> "MonoCut":
             supervisions=supervisions_vp,
         )
 
+    def reverb_rir(
+        self,
+        rir_cut: "MonoCut",
+        shift_output: bool = True,
```

Comment:
Note: if you set `shift_output` to False, it might affect the cuts duration (and maybe the offset). Not sure if this is worth supporting, maybe it's better to remove this option from cuts and let the users reverberate recordings first if they don't want to shift output? If you decide to go this way, please add a note in the docs.

Reply:
Okay I'll remove the option and only have it so the duration remains consistent.